packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -452,6 +449,34 @@ const toggleOrientation = async ( driver ) => {
 	}
 };
 
+const isEditorVisible = async ( driver ) => {
+	const blockLocator = isAndroid()
+		? `//android.widget.EditText[contains(@content-desc, "Post title")]`
+		: `(//XCUIElementTypeScrollView/XCUIElementTypeOther/XCUIElementTypeOther[contains(@name, "Post title")])`;
```

Comment:
I'm wondering if we should rename `blockLocator` to `postTitleLocator` as the query is related to the post title element, wdyt?